agimus_controller/agimus_controller/trajectories/sine_wave_cartesian_space.py

```diff
@@ -15,48 +18,117 @@ class SinusWaveCartesianSpace(TrajectoryBase):
 
     def __init__(
         self,
-        sine_wave_params,
+        sine_wave_params: SinWaveParams,
         ee_frame_name,
         w_q,
         w_qdot,
         w_qddot,
         w_robot_effort,
         w_pose,
+        mask=6 * [True],
```

Comment:
I'm surprised the linter does not complain about this. Default arguments in Python should be immutable. Two solutions here:
1. use a tuple instead of a list.
2. make the default value be None and, in the function body, set `self.mask` to a list of 6 `True`.